apps/journeys-admin/pages/reports/visitors.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactElement } from 'react'
+import {
+  AuthAction,
+  useAuthUser,
+  withAuthUser,
+  withAuthUserTokenSSR
+} from 'next-firebase-auth'
+import { NextSeo } from 'next-seo'
+import { useRouter } from 'next/router'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { useTranslation } from 'react-i18next'
+import { getLaunchDarklyClient } from '@core/shared/ui/getLaunchDarklyClient'
+import { PageWrapper } from '../../src/components/PageWrapper'
+import i18nConfig from '../../next-i18next.config'
+
+function VisitorsReportsPage(): ReactElement {
```

Comment:
consistency with route

```suggestion
function ReportsVisitorsPage(): ReactElement {
```